pl_examples/hydra_examples/conf/scheduler.py

```diff
@@ -0,0 +1,133 @@
+from dataclasses import dataclass
+
+from hydra.core.config_store import ConfigStore
+from hydra.types import ObjectConf
+from typing import List
+
+
+@dataclass
+class CosineConf:
+    T_max: int = 100
+    eta_min: float = 0
+    last_epoch: int = -1
+
+
+@dataclass
+class CosineWarmConf:
+    T_0: int = 10
+    T_mult: int = 1
+    eta_min: float = 0
+    last_epoch: int = -1
+
+
+# TODO Fix Cyclic
```

Comment:
What is this TODO referring to exactly?

Reply:
I have to go back and fix a few things about each of the schedulers and optimizers. Many have union types which are unsupported in Hydra at the moment, so I need to go back and make a few params Any so they can except List or ints for example.